src/bundle/Resources/views/admin/search/search_form.html.twig

```diff
@@ -12,48 +12,86 @@
-            <option class="ez-filters__option ez-filters__option--hidden" data-default="{{ 'search.any.content.type'|trans|desc('Any content type') }}" value="">{{ 'search.any.content.type'|trans|desc('Any content type') }}</option>
-        </select>
-        {{ form_widget(form.content_types, {'attr': {'class': 'ez-filters__select'}}) }}
-    </div>
-    {% if form.section is defined %}
-        <div class="ez-filters__item ez-filters__item--section">
-            <label class="ez-filters__item-label">{{ 'search.section'|trans|desc('Section:') }}</label>
-            {{ form_widget(form.section, {'attr': {'class': 'ez-filters__select'}}) }}
+<div class="ez-filters mt-4 ml-4 pt-2 {% if not filters_expanded %} ez-filters--collapsed{% endif %}">
+    <div class="ez-filters__row">
+        <div class="ez-filters__item ez-filters__item--content-type">
+            <label class="ez-filters__item-label">{{ 'search.content.type'|trans|desc('Content Type:') }}</label>
+            <select class="form-control ez-filters__select ez-filters__select--content-type" hidden="true">
+                <option class="ez-filters__option ez-filters__option--hidden" data-default="{{ 'search.any.content.type'|trans|desc('Any content type') }}" value="">{{ 'search.any.content.type'|trans|desc('Any content type') }}</option>
+            </select>
+            {{ form_widget(form.content_types, {'attr': {'class': 'ez-filters__select'}}) }}
         </div>
-    {% endif %}
-    <div class="ez-filters__item ez-filters__item--modified">
-        <label class="ez-filters__item-label">{{ 'search.last.modified'|trans|desc('Last modified:') }}</label>
-        {{ form_widget(form.last_modified_select, {'attr': {'class': 'ez-filters__select', 'data-target-selector': '.ez-modal--select-modified-range'}}) }}
     </div>
-    <div class="ez-filters__item ez-filters__item--created">
-        <label class="ez-filters__item-label">{{ 'search.created'|trans|desc('Created:') }}</label>
-        {{ form_widget(form.created_select, {'attr': {'class': 'ez-filters__select', 'data-target-selector': '.ez-modal--select-created-range'}}) }}
+    <div class="ez-filters__row">
+        {% if form.section is defined %}
+            <div class="ez-filters__item ez-filters__item--section">
+                <label class="ez-filters__item-label">{{ 'search.section'|trans|desc('Section:') }}</label>
+                {{ form_widget(form.section, {'attr': {'class': 'ez-filters__select'}}) }}
+            </div>
+        {% endif %}
+        <div class="ez-filters__item ez-filters__item--subtree">
+            <label class="ez-filters__item-label">{{ 'search.subtree'|trans|desc('Subtree:') }}</label>
+            {% set subtree = form.vars.data.subtree %}
+            <div>
+                <button class="btn btn-secondary ez-btn--udw-select-location"
+                    type="button"
+                    {% if form.subtree.vars.value is not empty %} hidden="true" {% endif %}
```

Comment:
Just `hidden`

Reply:
I actually prefer `hidden` but I thought that we have convention to use `hidden="true"`. Also, we have many examples of: `{'attr': {'hidden': 'hidden', `. So I'm confused a little bit. Do we have some convention? Or maybe this `{'attr': {'hidden': 'hidden', ` is different from just `hidden`, because the second one is in tag?

Reply:
When using Twig variables and object you can't set the attribute without value. But when writing in HTML you don't have to write `hidden="hidden"` because `hidden` is enough.

Reply:
AFAIK you can have empty attribute in twig: `{'attr': {'hidden': null}}`. But I think we use `{'attr': {'hidden': 'hidden'}}` so that it is more obvious, because `'hidden': null` may suggest that attribute won't be set. 

But here we have HTML, so I will change this to just `hidden`.